pulumi/grapl/__main__.py

```diff
@@ -156,41 +171,13 @@ def main() -> None:
         kafka_endpoint = "LOCAL_GRAPL_REPLACE_IP:19092"  # intentionally not 29092
         redis_endpoint = "redis://LOCAL_GRAPL_REPLACE_IP:6379"
 
-        grapl_core_job_vars = pulumi.Output.all(
-            analyzer_bucket=analyzers_bucket.bucket,
-            analyzer_dispatched_bucket=dispatched_analyzer_emitter.bucket.bucket,
-            analyzer_dispatcher_queue=analyzer_dispatcher_queue.main_queue_url,
-            analyzer_executor_queue=analyzer_executor_queue.main_queue_url,
-            analyzer_matched_subgraphs_bucket=analyzer_matched_emitter.bucket.bucket,
-            analyzer_dispatcher_dead_letter_queue=analyzer_dispatcher_queue.dead_letter_queue_url,
-            aws_access_key_id=aws.config.access_key,
-            aws_access_key_secret=aws.config.secret_key,
-            engagement_creator_queue=engagement_creator_queue.main_queue_url,
-            graph_merger_queue=graph_merger_queue.main_queue_url,
-            graph_merger_dead_letter_queue=graph_merger_queue.dead_letter_queue_url,
-            session_table_name=dynamodb_tables.dynamic_session_table.name,
-            schema_properties_table_name=dynamodb_tables.schema_properties_table.name,
-            schema_table_name=dynamodb_tables.schema_table.name,
-            model_plugins_bucket=model_plugins_bucket.bucket,
-            node_identifier_queue=node_identifier_queue.main_queue_url,
-            node_identifier_dead_letter_queue=node_identifier_queue.dead_letter_queue_url,
-            node_identifier_retry_queue=node_identifier_queue.retry_queue_url,
-            osquery_generator_queue=osquery_generator_queue.main_queue_url,
-            osquery_generator_dead_letter_queue=osquery_generator_queue.dead_letter_queue_url,
-            subgraphs_merged_bucket=subgraphs_merged_emitter.bucket,
-            subgraphs_generated_bucket=subgraphs_generated_emitter.bucket,
-            sysmon_generator_queue=sysmon_generator_queue.main_queue_url,
-            sysmon_generator_dead_letter_queue=sysmon_generator_queue.dead_letter_queue_url,
-            unid_subgraphs_generated_bucket=unid_subgraphs_generated_emitter.bucket,
-            user_auth_table=dynamodb_tables.user_auth_table.name,
-            user_session_table=dynamodb_tables.user_session_table.name,
-        ).apply(
+        grapl_core_job_vars = pulumi.Output.all(**nomad_inputs,).apply(
             lambda inputs: {
                 # This is a special directive to our HCL file that tells it to use Localstack
                 "_aws_endpoint": aws_endpoint,
-                "deployment_name": config.DEPLOYMENT_NAME,
+                "aws_access_key_id": aws.config.access_key,
+                "aws_access_key_secret": aws.config.secret_key,
                 "grapl_test_user_name": config.GRAPL_TEST_USER_NAME,
-                "aws_region": aws.get_region().name,
                 "_redis_endpoint": redis_endpoint,
```

Comment:
Why does this have an underscore? Is this also somehow applicable only for Localstack?

Reply:
Is the idea that everything in this map, aside from the `inputs` values, is _only_ for Localstack use?

Reply:
To the latter comment, yes these are localstack specific values

Reply:
To the first, I've added a comment to this effect, but this is basically a hint that we should be using the local. version of the env var instead of the var. version in the hcl (because we need the mutated version of this)